src/Essentials/src/Email/Email.shared.cs

```diff
@@ -60,31 +60,29 @@ public Task ComposeAsync(EmailMessage? message)
 			return PlatformComposeAsync(message);
 		}
 
-		static string GetMailToUri(EmailMessage message)
-		{
-			if (message != null && message.BodyFormat != EmailBodyFormat.PlainText)
-				throw new FeatureNotSupportedException("Only EmailBodyFormat.PlainText is supported if no email account is set up.");
+		static string GetMailToUri(EmailMessage message) =>
+			"mailto:?" + string.Join("&", Parameters(message));
 
-			var parts = new List<string>();
-			if (!string.IsNullOrEmpty(message?.Body))
-				parts.Add("body=" + Uri.EscapeDataString(message!.Body));
-			if (!string.IsNullOrEmpty(message?.Subject))
-				parts.Add("subject=" + Uri.EscapeDataString(message!.Subject));
-			if (message?.Cc?.Count > 0)
-				parts.Add("cc=" + Uri.EscapeDataString(string.Join(",", message.Cc)));
-			if (message?.Bcc?.Count > 0)
-				parts.Add("bcc=" + Uri.EscapeDataString(string.Join(",", message.Bcc)));
+		static IEnumerable<string> Parameters(EmailMessage message)
+		{
+			if (message.To?.Any() == true)
```

Comment:
Using LINQ and Any() is not so performant. Maybe it is now, but we can also just use ?.Count > 0 which we know is fast.

Reply:
Nobody is composing emails in a tight loop, I think we can let this go and worry about the performance characteristics of this another day 🙂

Reply:
Up to you whether you want me to use Count() > 0 or Any(). I have no strong feelings about this one.

Reply:
Fixed. Can you have another look?